### PR TITLE
Improve .Scratch entry

### PR DESCRIPTION
**Pull request by regisphilibert**
Hi Guys,

I finally set myself to improve the Scratch page with concised but yet helpful examples. I think the previous ones failed to include less technical users.

Feel free to improve the methods' one line descriptions, I know @bep is really good at that!

I thought about plugging my scratch post in there, but not sure it is really needed now.

cc/ @budparr @kaushalmodi

### CODE REVIEW

**Comment by bep** [comment]
>I thought about plugging my scratch post in there, but not sure it is really needed now.

Scratch that ... :-)

File: content/functions/scratch.md [hunk 1]
```diff
@@ -20,66 +20,90 @@ draft: false
...
+
+```go-html-template
+{{ .Scratch.Delete "greetings" }}
 ```
 
-{{% note %}}
-The examples above uses the special `$` variable, which refers to the top-level node. This is the behavior you most likely want, and will help remove some confusion when using `Scratch` inside page range loops -- and you start inadvertently calling the wrong `Scratch`. But there may be use cases for `{{ .Scratch.Add "key" "some value" }}`.
-{{% /note %}}
+## Scope
+The scope of the backing data is global for the given `Page` or `Shortcode`, and spans partial and shortcode includes.
```
Comment (kaushalmodi) [inline content/functions/scratch.md]: Do you also want to mention the use case for "root level" scope `$`? I see that this commit completely removes that mention.

Comment (regisphilibert) [reply content/functions/scratch.md]: Yes, I always found it more confusing than helping. $ is universal to any page context method. I don't think we should focus too much on that here. But I may be wrong.

Comment (bep) [reply content/functions/scratch.md]: I agree with @regisphilibert -- I think these gotchas should be in one (or two: faq also maybe) place, and not sprinkled around everywhere. It removes focus.

**Comment by regisphilibert** [comment]
Alright, I added the ref to the post.

**Pull request merged**